Demo/Demo/Features/CardDemoViewController.swift

```diff
@@ -0,0 +1,106 @@
+import UIKit
+
+class CardDemoViewController: FeatureBaseViewController {
+    // TODO: Validate card fields are filled out
+    // TODO: Add proper spacing / styling / max characters to card fields
+    // TODO: Animate pay button to indicate loading
+    // TODO: Disable pay button until fields are filled out
+
+    private enum Constants {
+        static let stackViewSpacing: CGFloat = 16
+        static let layoutSpacing: CGFloat = 16
+        static let textFieldHeight: CGFloat = 40
+    }
+
+    let mainStackView: UIStackView = {
```

Comment:
`mainStackView` and `secondaryStackView` are a bit confusing - maybe better to rename them based on what they contain? Like `cardFormStackView` and `expirationCVVStackView` or something.